Resources/public/js/views/services/ez-locationviewviewservice.js

```diff
@@ -43,6 +44,110 @@ YUI.add('ez-locationviewviewservice', function (Y) {
         },
 
         /**
+         * _sendContentToTrashConfirmBox event handler on send to trash button
+         *
+         * @method _sendContentToTrashConfirmBox
+         * @protected
+         * @param {Object} e event facade of the sendToTrashAction event
+         */
+        _sendContentToTrashConfirmBox: function (e) {
+            e.preventDefault();
+            this.fire('confirmBoxOpen', {
+                config: {
+                    title: "Are you sure you want to send this content to trash?",
+                    confirmHandler: Y.bind(function () {
+                        this._sendToTrash(e.content);
+                    }, this)
+                },
+            });
+        },
+
+        /**
+         * Sends location to trash, triggering loading parent location and notifications
+         *
+         * @method _sendToTrash
+         * @protected
+         * @param {Object} content
+         */
+        _sendToTrash: function (content) {
+            var options,
+                that = this,
+                location = this.get('location'),
+                locationId = location.get('id'),
+                path = this.get('path'),
+                contentName = content.get('name');
+
+            this._notify(
+                'Sending "' + contentName + '" to Trash',
+                'send-to-trash-' + locationId,
+                'started',
+                0
+            );
+
+            options = {
+                parentLocation: path[path.length - 1].location,
+                contentName: contentName
+            };
+
+            location.trash({api: this.get('capi')}, Y.bind(that._afterSendToTrashCallback, that, options));
+        },
+
+        /**
+         * Send to trash callback triggering notifications and making app to navigate to parent location
+         *
+         * @method _afterSendToTrashCallback
+         * @protected
+         * @param {Object} options the options for sending to trash
```

Comment:
you should better describe what you expect in this object with something like:
```
* @param {Object} options
* @param {eZ.Location} options.parentLocation
* @param {String} options.contentName
```
(or use individual parameters since options is supposed to contain only the parentLocation and the contentName).